src/main/java/org/folio/marccat/business/cataloguing/authority/AuthorityTagImpl.java

```diff
@@ -110,7 +106,30 @@ public Validation getValidation(Tag t, Session session) {
   public Set getValidEditableSubfields(int category) {
     // TODO It is an abstract class that should be implemented. At the moment this
     // function is not used.
-    return null;
+    return Collections.emptySet();
   }
 
+  public int getTagCategory(final Heading heading, Session session) {
+
+    try {
+      List<AuthorityCorrelation> correlations = daoCorrelation.getCategoryCorrelation(session, heading.getTag(),
```

Comment:
Are we certain that `getInd1()` and `getInd2()` won't return null?

Reply:
We think that  the indicator never will be null, but I have checked it.